xla/service/gpu/runtime/command_buffer_cmd.h

```diff
@@ -769,7 +768,7 @@ class CuDnnCmd : public TracedCommandBufferCmd {
 
  private:
   std::vector<BufferAllocation::Slice> args_;
-  const se::dnn::DnnGraph& graph_;
+  const std::unique_ptr<se::dnn::DnnGraph>& graph_;
```

Comment:
Instead of taking a reference you can use `std::shared_ptr`, the fact that Thunks stay alive is mostly accidental because of the workaround for cupti+cuda graphs bugs, this will go away once CUDA fix will land

Reply:
Done.